functions/bouncer/src/validate/stripe/__tests__/data/checkoutCompleteEvent.json

```diff
@@ -0,0 +1,74 @@
+{
```

Comment:
it looks scrubbed, but just in case it's a, make sure it's scrubbed.

Reply:
verified scrubbed